### ISSUE TITLE
Strange formatting of long xor statement within if

### ISSUE THREAD

**Issue by grimdug** [perltidy/perltidy#19]
```
$ cat t.pl
if ( mysub(1) == 0 xor mysub(2) == 0)  { 1; }

$ ~/Perl-Tidy-20200110/perltidy-20200110.pl -l=30 t.pl

$ cat t.pl.tdy 
if ( mysub(1) ==
    0 xor mysub(2) == 0 )
{
    1;
}
```

Should do a line break between "0" and "xor" in my opinion ...

**Comment by shancock9** [comment]
Thanks for the note. The poor choice of line break here is controlled by tables internal to perltidy.  I can fix this in the next release, by treating breaks on 'xor' similar to 'or' and '||', but unfortunately there is no user control (other than forcing a break with a side comment after the 0).  Steve.

**Comment by shancock9** [comment]
this is fixed in v 20200619
